lib/chef/client.rb

```diff
@@ -721,7 +676,6 @@ def converge(run_context)
           events.converge_complete
         rescue Exception => e
           events.converge_failed(e)
-          raise e if Chef::Config[:audit_mode] == :disabled
```

Comment:
This bit seems odd to just remove

Reply:
(the next line does go away then since raise always raises and the `converge_exception = e` becomes dead code.